components/Layout.js

```diff
@@ -1,10 +1,29 @@
+import Link from "next/link";
 import Header from "./Header";
```

Comment:
Just for readability: try to do the react and next imports first, then 3rd party components (styled), then your own...

Reply:
Thanks Ben, I'll update it in my next commit